website/scripts/linkchecker.sh

```diff
@@ -31,13 +32,40 @@ GREP_STATUS=$?
 
 if [[ $GREP_STATUS != 0 ]]; then
   # examine wget.log
-  BROKEN_MSG="broken link\."
+  BROKEN_MSG="^Found.*broken link"
   BROKEN_MSG_LINE=$(grep -n "${BROKEN_MSG}" ${WGET_LOG})
   LN=$(echo "${BROKEN_MSG_LINE}" | cut -f1 -d:)
-  tail "+${LN}" "$WGET_LOG"
-  exit 1
+  LINES=$(tail "+${LN}" "${WGET_LOG}")
+  BAD_LINKS=""
+  COUNT=0
+  # filter out broken links with prefix ``localhost:15532``
```

Comment:
this seems like it would filter valid links to pages we want to check. Don't we want to filter out links to the tracker and heron ui on localhost?